mobilevit/models/conv_block.py

```diff
@@ -0,0 +1,74 @@
+import tensorflow as tf
+from keras.applications import imagenet_utils
+from tensorflow.keras import layers
+
+
+def conv_block(input_layer, num_filters: int = 16, strides: int = 2):
+    """
+    3x3 Convolutional Stem Stage.
+    Args:
+        input_layer: input tensor
+        num_filters (int): number of filters in the convolutional layer
+        stries (int): stride of the convolutional layer
+    Returns:
+        output tensor
+    """
+    conv_1 = layers.Conv2D(
+        filters=num_filters,
+        kernel_size=(3, 3),
+        strides=strides,
+        padding="same",
+    )(input_layer)
```

Comment:
Makes sense to give meaningful names to the layers.

Reply:
Cool

Reply:
> Makes sense to give meaningful names to the layers.

Are you talking about `conv_1` or this

```
conv_1 = layers.Conv2D(
         filters=num_filters,
         kernel_size=(3, 3),
         strides=strides,
         padding="same",
         name="something", // Like this
     )(input_layer)
```

Reply:
The `name` argument.